dashboard/app/models/stage.rb

```diff
@@ -31,6 +31,7 @@ class Stage < ActiveRecord::Base
   has_many :script_levels, -> {order('position ASC')}, inverse_of: :stage
   has_one :plc_learning_module, class_name: 'Plc::LearningModule', inverse_of: :stage, dependent: :destroy
   belongs_to :script, inverse_of: :stages
+  belongs_to :lesson_group
```

Comment:
In order for this to work, the `stages` table needs to have a `lesson_group_id` on it, right? I don't see that in the migration or the model comments. not sure if I am missing something here, but is this perhaps something that you added in your local DB in a past version of the migrations but which didn't make it into this PR?

Reply:
Its in this PR: https://github.com/code-dot-org/code-dot-org/pull/34043 which I can just merge into this PR if that is better?

Reply:
thank you for pointing that out, I had forgotten about that PR! I think I am confused about the order of the PRs -- it looks like #34043 was sent out earlier than this one, but this one looks like #34043 builds on it. assuming you plan to merge this one first, then yes, this PR should either contain the migration which adds stage.lesson_group_id, or this PR should not add the association yet.